Dockerfile

```diff
@@ -1,11 +1,11 @@
-FROM openshift/ruby-20-centos7
+FROM centos/ruby-22-centos7
```

Comment:
Our most senarios are doing s2i build using this repo with rhel7 image which will replace images in Dockerfile.